pymc3/plots/__init__.py

```diff
@@ -53,6 +52,38 @@ def wrapped(*args, **kwargs):
 densityplot = map_args(az.plot_density)
 pairplot = map_args(az.plot_pair)
 
+# addition arg mapping for compare plot
+swaps = [
+    ('varnames', 'var_names')
+]
+@functools.wraps(az.plot_compare)
+def compareplot(*args, **kwargs):
+    args = list(args)
+    comp_df = args[0].copy()
```

Comment:
This assumes that comp_df is passed as the 0th arg. Does this cover the case where comp_df is passed a kwarg?